### PR TITLE
State in forms

### PR DESCRIPTION
**Pull request by Brolyus**
Added the user story "Users can add a custom vehicle" to the merged website (from presentation branch). Also, now all the choice from users goes to the state of game.js. Work is still in progress on game.js file as we implement the "calculate" function

### CODE REVIEW

File: astra/src/components/Game.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    speed: "10"
+  }
+];
+
+class Game extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      planet_options: planetList.map(option => option.englishName),
+      vehicle_options: vehicleList.map(option => option.vehicle),
```
Comment (rocambille) [inline astra/src/components/Game.js:53]: snake_case à passer en camelCase, un peu partout ;)

**Review by rocambille — CHANGES_REQUESTED** [review]

File: astra/src/components/Game.js [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    vehicle: "Vélo",
+    speed: "10"
+  }
+];
+
+class Game extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      planet_options: planetList.map(option => option.englishName),
```
Comment (ghost) [inline astra/src/components/Game.js:52]: use componentDidMount to manage those data

File: astra/src/components/Game.js [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+    event.preventDefault();
+    const getPlanetsProperties = [...planetList].map(propertie => {
+      console.log(propertie)
+    })
+
+    // Afficher dans le result: Distance / Temps de trajet / Vitesse / Véhicule
+    const result = this.state.depart - this.state.arrival;
+
+    if (result < 0) {
+      console.log(-result);
```
Comment (ghost) [inline astra/src/components/Game.js:95]: No need to keep console.log() in dev

File: astra/src/components/Game.js [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+      return result;
+    }
+  }
+
+  handleSubmitNewVehicle(event) {
+    vehicleList[vehicleList.length] = {
+      vehicle: this.state.customVehicle,
+      speed: this.state.customSpeed
+    };
+    console.log(vehicleList);
```
Comment (ghost) [inline astra/src/components/Game.js:108]: another console log

File: astra/src/components/calculator.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
+let result = 0
```
Comment (ghost) [inline astra/src/components/calculator.js:1]: When declaring a function, set all of its value inside it, not outside

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by Brolyus** [comment]
Fixed in PR 9

**Pull request closed without merging**